src/panels/lovelace/views/hui-view-container.ts

```diff
@@ -89,49 +72,13 @@ class HuiViewContainer extends LitElement {
     if (this.hass) {
       applyThemesOnElement(this, this.hass?.themes, this.theme);
     }
-
-    const computedStyles = getComputedStyle(this);
-    const themeBackground = computedStyles.getPropertyValue(
-      "--lovelace-background"
-    );
-
-    const fixedBackground = this._isFixedBackground(
-      this.background || themeBackground
-    );
-    const viewBackground = this._computeBackgroundProperty(this.background);
-    this.toggleAttribute("fixed-background", fixedBackground);
-    this.style.setProperty("--view-background", viewBackground);
   }
 
   static get styles(): CSSResultGroup {
     return css`
       :host {
         display: relative;
       }
-      /* Fixed background hack for Safari iOS */
```

Comment:
Since your removed this, does it still work on iOS?

Reply:
It was introduced in https://github.com/home-assistant/frontend/pull/22531, I've added it back but can't test on iOS.